README.md

```diff
@@ -80,19 +80,19 @@ table = (;a, b, c, d)
 # corner plot of original table
 table |> corner
 ```
-![original](docs/original.png)
+![original](imgs/original.png)
```

Comment:
Please move these other images there as well temporarily. They will go away after the documentation is built. The README will be shortened to include just the logo.

Reply:
OK

Reply:
Made in [44a7d1b](https://github.com/JuliaML/TableTransforms.jl/pull/46/commits/44a7d1b3391c77a08f08f9b597d00bc07eb40b97)